_data/chains/eip155-56288.json

```diff
@@ -0,0 +1,27 @@
+{
+  "name": "Boba BNB Mainnet",
+  "chain": "Boba BNB Mainnet",
+  "rpc": [
+    "https://bnb.boba.network",
+    "wss://wss.bnb.boba.network",
+    "https://replica.bnb.boba.network",
+    "wss://replica-wss.bnb.boba.network"
+  ],
+  "faucets": [],
+  "nativeCurrency": {
+    "name": "Boba Token",
+    "symbol": "BOBA",
+    "decimals": 18
+  },
+  "infoURL": "https://boba.network",
+  "shortName": "BobaBnb",
```

Comment:
```suggestion
  "shortName": "BobaBnbOld",
```

Reply:
chain id 56288 is the "new" chain. fixed